api/lib/mno_enterprise/concerns/controllers/jpi/v1/admin/organizations_controller.rb

```diff
@@ -26,6 +26,10 @@ module MnoEnterprise::Concerns::Controllers::Jpi::V1::Admin::OrganizationsContro
   #==================================================================
   # GET /mnoe/jpi/v1/admin/organizations
   def index
+    # When fetching multiple organizations attached to one user, we must fetch the user's role for each organization.
+    where = params["where"]
+    @user = MnoEnterprise::User.find_one(params["where"]["users.id"], :orga_relations) if where && where["users.id"]
```

Comment:
Can we refactor this like this maybe? 
```
@user = MnoEnterprise::User.find_one(params["where"]["users.id"], :orga_relations) if params.dig('where', 'users.id')
```
Then the assignment on line 30 can be removed.